CMakeLists.txt

```diff
@@ -375,7 +377,7 @@ endif()
 
 message("FINAL CMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}")
 
-if(${VELOX_ENABLE_GPU})
+if(VELOX_ENABLE_GPU)
```

Comment:
I have a todo to write a cmake style guide and this would be a point in it ^^ :+1: